### PR TITLE
[website]rest api documentation patch for GC and ReadonlyState

### PR DESCRIPTION
**Pull request by zwOvO**
### Motivation
Improve the Http Rest API documentation

### CODE REVIEW

**Comment by zwOvO** [comment]
#3845

File: site3/website/docs/admin/http.md [hunk 1]
```diff
@@ -462,6 +468,38 @@ Currently all the HTTP endpoints could be divided into these 5 components:
        }
       ```
 
+### Endpoint: /api/v1/bookie/state/readonly
+1. Method: GET
+    * Description: Get bookie readOnly state.
+    * Response:  
+    
+        | Code   | Description |
```
Comment (horizonzy) [inline site3/website/docs/admin/http.md:476]: The response should contains the `readOnly` property

Comment (zwOvO) [reply site3/website/docs/admin/http.md]: OK

**Comment by zwOvO** [comment]
@horizonzy @jvrao @ivankelly @fcuny PTAL

**Comment by zwOvO** [comment]
@hangc0276 @horizonzy @eolivelli PTAL

**Review by hangc0276 — APPROVED** [review]

**Pull request closed without merging**